### ISSUE TITLE
The App.db arent restoring correctly

### ISSUE THREAD

**Issue by fabrebatalla18** [stooged/Db_Restore#1]
When you restore App.db (from the internal disk) the restoration process is interrupted before the forced restart and the incomplete file is incomplete, maybe the solution would be to add a delay?

**Comment by RetroGamer74** [comment]
That's correct. According to my test, I did an internal backup using Database Only Backup payload. A new set of files were created at system_data/priv/mms as for example app.db_backup, addcont.db_backup and so on. All of the files created had same size than the original. I did a restore, and after the reboot, my home screen had nothing but PS4 common apps. I took a look to the folder again thru FTP, and all the files restored were exactly the same size but app.db which its size was different than the backup file. So my guess is maybe because of the forced reboot, the flush to disk of the copy command was not complete. This behaviour was tested on several PS4, not only mine.

Best regards.

**Comment by stooged** [comment]
It has a 8 second delay before the reboot.
I will have to test it when i get some time.

**Comment by RetroGamer74** [comment]
Hi stooged. I did a video in my channel explaining this. It is in spanish but you can use real time translation. I think, the reboot happens very fast. I'm pretty sure, I didn't feel to wait that time.

If you're interested in the video, let me know it, and I will give you the link. I don't want to do spam :)

Cheers.
